test/inlining/InlineCallbacks.js

```diff
@@ -0,0 +1,23 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------

```

Comment:
Are there other configurations that might be worth testing? Maybe passing the callback through multiple layers like `function Dispatch2(f) { Dispatch(f); }` or a callback that calls things like `function Dispatch3(f) { f(Foo); }` where we call `Dispatch3(Dispatch)` after the warmup?

Reply:
or maybe a case where the callback isn't the first param

---
In reply to: [185833307](https://github.com/Microsoft/ChakraCore/pull/5081#discussion_r185833307) [](ancestors = 185833307)